lib/process_settings/testing/helpers.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'process_settings/monitor'
+require 'process_settings/settings'
+require 'process_settings/target_and_settings'
+
+require 'process_settings/testing/monitor'
+
+module ProcessSettings
+  module Testing
+    module Helpers
+      extend self
+
+      TRUE_TARGET = ProcessSettings::Target.new({})
+
+      # Adds the given settings_hash as an override at the end of the process_settings array, with default targeting (true).
+      # Therefore this will override these settings while leaving others alone.
+      #
+      # @param [Hash] settings_hash
+      #
+      # @return none
+      def stub_process_settings(settings_hash)
+        new_target_and_settings = ProcessSettings::TargetAndSettings.new(
+          '<test_override>',
+          TRUE_TARGET,
+          ProcessSettings::Settings.new(settings_hash.deep_stringify_keys)
+        )
+
+        new_process_settings = [
+          *default_process_settings.statically_targeted_settings,
+          new_target_and_settings
+        ]
+
+        ProcessSettings::Monitor.instance = ProcessSettings::Testing::Monitor.new(
+          new_process_settings,
+          logger: default_process_settings.logger
+        )
+      end
+
+      # Returns the default process settings monitor instance that loads the combined_process_settings.yml
```

Comment:
My intention was to put this back into the Monitor class, so that `def instance` would become
```
@instance ||= default_instance
```
And the current body of `def instance` becomes `def default_instance`.

Reply:
I'd like to defer to you on how you'd like to do this.

Reply:
Ditto on the PR off the PR.

Reply:
I think I have this as part of the refactor